### DISCUSSION TITLE
The image is not accessible

### DISCUSSION THREAD

**Discussion by SagitTariuse** [chevereto/chevereto#34] [Q&A]
When I use picgo+chevereto, I can upload the image using ip+ port normally, but when I use the domain name plus port, I can't, and when I use the domain name plus port to access my image is displayed as the visitor upload
![PR7DVWWNQN%XH%P`VNP45PX](https://github.com/chevereto/chevereto/assets/81310072/bf8b1f8e-ca13-449a-9e49-6df489ab12d1)

![SV(GW)2`K_2$84TL}SXG3Z5](https://github.com/chevereto/chevereto/assets/81310072/1ad7a7ef-e776-4ff1-88bd-8a852724ef43)
Above is the form I use domain name plus port
Below I will use the ip+ port form
![image](https://github.com/chevereto/chevereto/assets/81310072/d815c7cf-4467-4f8b-9a62-e64c27c0ab55)
Can view normally

**Comment by rodber** [comment]
I don't understand how this is an issue for us?

Chevereto requires to configure some variables which will be used to build the URLs but Chevereto isn't in charge of resolving these URLs.
